js/gate.js

```diff
@@ -4102,7 +4102,11 @@ module.exports = class gate extends Exchange {
          * @returns {[object]} a list of [position structure]{@link https://docs.ccxt.com/en/latest/manual.html#position-structure}
          */
         await this.loadMarkets ();
+        symbols = this.marketSymbols (symbols);
         const [ type, query ] = this.handleMarketTypeAndParams ('fetchPositions', undefined, params);
+        if (type !== 'swap' && type !== 'future') {
```

Comment:
@pcriadoperez if the user provides one or more symbols that should be used to infer the type and currently we're ignoring it